workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedGetAppPage/PurchasedGetAppPage.tsx

```diff
@@ -0,0 +1,600 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import DropDown from '@clayui/drop-down';
+import ClayForm, {ClayCheckbox, ClayInput} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClayLink from '@clayui/link';
+import {InputHTMLAttributes, useEffect, useMemo, useState} from 'react';
+import {useForm} from 'react-hook-form';
+import {z} from 'zod';
+
+import {Header} from '../../components/Header/Header';
+import BaseWrapper from '../../components/Input/base/BaseWrapper';
+import zodSchema, {zodResolver} from '../../schema/zod';
+import {
+	getAccountInfo,
+	getListTypeDefinitionByExternalReferenceCode,
+	getProductById,
+	getProductSKU,
+	getUserAccount,
+	postAccountByERCUserAccountByERC,
+} from '../../utils/api';
+
+import './PurchasedGetAppPage.scss';
+
+import ClaySticker from '@clayui/sticker';
+
+import emptyPictureIcon from '../../assets/icons/avatar.svg';
+import {Footer} from '../../components/Footer/Footer';
+import Select from '../../components/Select/Select';
+import fetcher from '../../services/fetcher';
+import CreatedProjectCard from './CreatedProjectCard';
+import PurchasedGetAppAccountSelection from './PurchasedGetAppAccountSelection';
+import {getPhones} from './PurchasedGetAppPageUtil';
+
+type Steps = {
+	page:
+		| 'accountCreation'
+		| 'accountSelection'
+		| 'projectCreated'
+		| 'initialStep';
+};
+
+const accountTypes = {
+	BUSINESS: 'business',
+	PERSON: 'person',
+};
+
+export type UserForm = z.infer<typeof zodSchema.accountCreator>;
+
+type InputProps = {
+	boldLabel?: boolean;
+	className?: string;
+	description?: string;
+	disabled?: boolean;
+	errors?: any;
+	id?: string;
+	label?: string;
+	name: string;
+	options?: {label: string; value: string} | [];
+	register?: any;
+	required?: boolean;
+	type?: string;
+} & InputHTMLAttributes<HTMLInputElement>;
```

Comment:
Why don't we componentize this? We have the same code above.

Reply:
Hey, why do we already have a Component INput and we can't change the other component, otherwise it will break several other applications 😞